R/zzz.r

```diff
@@ -1,4 +1,8 @@
 .onLoad <- function(libname, pkgname) {
+  mutate <<- set_collector(mutate)
+  summarise <<- set_collector(summarise)
+  summarize <<- set_collector(summarize)
+
```

Comment:
once in a package we can do this in place from outside, and we can easily have a function to reverse the operation, very similar to untrace()